vivisect/__init__.py

```diff
@@ -316,6 +316,12 @@ def getEmulator(self, **kwargs):
         emu = eclass(self, **kwargs)
         emu.setEndian(self.getEndian())
 
+        # PowerPC special case, if there are memory maps defined propagate those
+        # to the new emulator
+        maps = self.getMeta('PpcMemoryMaps')
+        if maps is not None:
+            emu.setVleMaps(maps)
+
         return emu
```

Comment:
I have not come up with a method. Because the emulators or architecture modules can be re-created on the fly there is no guarantee that any information stored or tracked at one point will persist to all future iterations of the emulator (or architecture module). However I am open to suggestions. My knowledge of the internals of vivisect are not nearly as extensive as yours.